src/sql/plot.py

```diff
@@ -124,7 +133,9 @@ def _between(conn, table, column, whislo, whishi, with_=None):
 
 # https://github.com/matplotlib/matplotlib/blob/b5ac96a8980fdb9e59c9fb649e0714d776e26701/lib/matplotlib/cbook/__init__.py
 @modify_exceptions
-def _boxplot_stats(conn, table, column, whis=1.5, autorange=False, with_=None):
+def _boxplot_stats(
+    conn, table, column, whis=1.5, autorange=False, with_=None, driver=None
```

Comment:
Why do we set `driver` as optional? Is there a possibility that it will be None? (I guess if there is no payload but in this case other stuff will break)

Reply:
design question. Can we create a function that returns the current driver? This way, we won't have to add an extra parameter and rely on the payload.

Reply:
Yeah, I think I missed that conn was already available in the function. Removed the `driver` parameter